src/profile/hooks/useProfileQuery.ts

```diff
@@ -29,6 +29,10 @@ type QueryReturnType = {
 export function useProfileQuery(props?: {
   onError: (queryError: ApolloError) => void;
 }): QueryReturnType {
+  const errorToTriggerRef = useRef<ApolloError | undefined>(undefined);
```

Comment:
A few lines above this is a comment about [a bug in apollo-client](https://github.com/apollographql/apollo-client/issues/5531) that had to be circumvented. That bug is claimed to be fixed in Apollo client version 3.5.0. Can the previously non-usable functionality now be used?

Reply:
I tried that, but there were complications:
- when `errorPolicy: 'all'` is used, the onError callback is not triggered, if there is a graphQL-error
- if errorPolicy has another value, it doesn't allow graphQL errors. Then the profile won't load, if user is logged in with weak authentication. In that case there is always a graphQL error present stating user don't have rights to load verified personal information.

So still have to use a this hack.  I'll change the comment about the bug

Reply:
OK. The comment wasn't yet changed in the changes currently present. Is it coming, or did you have second thoughts? Otherwise this is looking good.

Reply:
Just took some time to re-write and rebase :) Done now.

Reply:
Got code smells from trailing spaces in the comments. Updated again.